### DISCUSSION TITLE
use vue_flow; but Module parse failed: Unexpected token

### DISCUSSION THREAD

**Discussion by RelaxRick** [bcakmakoglu/vue-flow#1091] [Q&A] [answered]
![image](https://github.com/bcakmakoglu/vue-flow/assets/74636897/64a9fdbd-2888-41cc-b01c-a0fb15ea76c3)

    "@vue-flow/core": "^1.18.0",
    "vue": "3.0.11",
   Is there any solution? thanks

**Comment by bcakmakoglu** [comment, marked as answer]
You might want to check this discussion: https://github.com/bcakmakoglu/vue-flow/discussions/851
